### ISSUE TITLE
update readme with more comprehensive documentation

### ISSUE THREAD

**Issue by martinlippert** [eduk8s/eduk8s-vscode-helper#2]
since the example experiments page on the boot workshop is going away soon, we need to document all the different things that you can do with this extension inside of eduk8s workshops - with examples - so that workshop authors can use the doc to learn about the options and use them.

**Comment by BoykoAlex** [comment]
We should first integrate forked Initializr and Remote control extensions into a lower level of eduk8s environment and place documentation there... Maybe here https://github.com/eduk8s/pkgs-code-server or create a dedicated repository under eduk8s, have quay image for extensions and place the docs there...

**Comment by kdvolder** [comment]
We created a code-server samples where we moved our examples / explanation of the different remote control features for the time being:

https://github.com/eduk8s-tests/code-server-samples

I think we should reserve documentation in readme in vscode-helper repo to just documenting the rest api endpoints that we have implemented. Whereas code-server-samples can show how its used in eduk8s.

In the mean time @GrahamDumpleton has started adopting vscode-helper into eduk8s base image and I think he is designing the way to use this from eduk8s in a different way than our prototype. That probably means also that he will take care of documenting this in eduk8s main documentation?

I will take a pass though the readme in this repo to make sure all our endpoints are documented.
